src/plugins/kibana/public/visualize/styles/main.less

```diff
@@ -45,7 +53,15 @@
     .list-group {
       margin-bottom: 0;
 
-      .list-group-item {
+      .list-group-item .list-group-item-menu:hover {
```

Comment:
I see `list-group-menu-item`, but it looks like everywhere it is used `list-group-item` is already a class on the element. Seems like it could go right?